src/infra/AutocompleteServiceImpl.ts

```diff
@@ -70,8 +71,17 @@ export class AutocompleteServiceImpl implements AutocompleteService {
     };
 
     const response = 'token' in this.config
-      ? await this.httpService.get<ApiResponse>(this.universalEndpoint, queryParams, this.config.token)
-      : await this.httpService.get<ApiResponse>(this.universalEndpoint, queryParams);
+      ? await this.httpService.get<ApiResponse>(
+        this.universalEndpoint,
+        queryParams,
+        getClientSdk(request.customClientSdk),
+        this.config.token
```

Comment:
Here and below the indentation needs to be corrected. The function args should be further in by one tab, they should not be flush with `await`.